packages/suite/src/components/suite/NavigationBar/components/NavigationActions/index.tsx

```diff
@@ -15,6 +15,7 @@ import { NavSettings } from './components/NavSettings';
 import { variables } from '@trezor/components';
 import { NavBackends } from './components/NavBackends';
 import type { Route } from '@suite-types';
+import { useGuide } from '@suite/hooks/guide';
```

Comment:
`@guide-hooks` + move it above import type 😇

Reply:
Solved 😬😬😬